addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -71,7 +71,7 @@ export class ProjectTaskFormController extends FormController {
                         )
                     )}</span>`
                 ),
-                recordId: this.props.resId,
+                recordId: record.resId,
```

Comment:
this change is not needed since the editor team is working on the "Add Current History" button

Reply:
> this change is not needed since the editor team is working on the "Add Current History" button

My goal in adding this commit is to ensure that no issues arise during the check-testing phase. we will remove it during the review stage.what do you think?